packages/cli/src/commands/apps/create.ts

```diff
@@ -38,12 +36,14 @@ export default class AppCreateCommand extends InputOutputAPICommand<AppRequest,
 						await Promise.all(requests)
 					}
 				} else {
-					this.logger.error('Authorization is not applicable to web-hook SmartApps')
-					// eslint-disable-next-line no-process-exit
-					process.exit(1)
+					throw new CLIError('Authorization is not applicable to web-hook SmartApps')
```

Comment:
Webhook shouldn't be hyphenated.

Reply:
Fixed (this one and three others) and also capitalized `WebHook` to match on-line documentation.